src/patternfly/components/Card/card.scss

```diff
@@ -21,21 +21,50 @@
       box-shadow: var(--pf-c-card--m-hoverable--hover--BoxShadow);
     }
   }
+}
 
-  .pf-c-card__header,
-  .pf-c-card__body,
-  .pf-c-card__footer {
-    flex: 0 0 auto;
-    padding-right: var(--pf-c-card--child--PaddingRight);
-    padding-bottom: var(--pf-c-card--child--PaddingBottom);
-    padding-left: var(--pf-c-card--child--PaddingLeft);
+.pf-c-card__head {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
 
-    &:first-child {
-      padding-top: var(--pf-c-card--first-child--PaddingTop);
-    }
+  > :not(.pf-c-card__actions) {
+    flex: 1 1 auto;
+  }
+}
+
+.pf-c-card__actions {
```

Comment:
Should we support a modifier for adds space between 2 of more actions? Or can they just use a layout to create that space?

Reply:
yeah... what about doing both? That's what I was thinking about doing in https://github.com/patternfly/patternfly-next/issues/1704 - just create the element and say like `__actions > * + * { margin-left: something; }`, to create space between whatever you put in the actions element, then if you want the elements to wrap or break at a breakpoint or use custom spacing or whatever, use a flex layout instead.